### PR TITLE
Add indexes to Activity table

### PR DESCRIPTION
**Pull request by Prokyonn**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| License | MIT

#### What's in this PR?

Add missing indexes for better performance on the table.

### CODE REVIEW

File: src/Sulu/Bundle/ActivityBundle/Resources/config/doctrine/Activity.orm.xml [hunk 1]
```diff
@@ -26,5 +26,11 @@
         <field name="resourceSecurityContext" column="resourceSecurityContext" type="string" nullable="true" length="191"/>
         <field name="resourceSecurityObjectType" column="resourceSecurityObjectType" type="string" nullable="true" length="191"/>
         <field name="resourceSecurityObjectId" column="resourceSecurityObjectId" type="string" nullable="true" length="191"/>
+
+        <indexes>
+            <index name="timestamp_idx" columns="timestamp"/>
+            <index name="resourceKey_idx" columns="resourceKey"/>
+            <index name="resourceId_idx" columns="resourceId"/>
```
Comment (alexander-schranz) [inline src/Sulu/Bundle/ActivityBundle/Resources/config/doctrine/Activity.orm.xml:33]: Looks like the `resourceLocale` should also have an index: https://github.com/sulu/sulu/blob/e8a62f408dd15e1b89b9bfff63dda2d89cc3597e/src/Sulu/Bundle/ActivityBundle/UserInterface/Controller/ActivityController.php#L152

Comment (alexander-schranz) [reply src/Sulu/Bundle/ActivityBundle/Resources/config/doctrine/Activity.orm.xml:33]: `resourceSecurityContext`, `resourceSecurityObjectType`, `resourceSecurityObjectId` are also be filtered and should I think then have an index.

Comment (Prokyonn) [reply src/Sulu/Bundle/ActivityBundle/Resources/config/doctrine/Activity.orm.xml:33]: @alexander-schranz fixed

**Review by alexander-schranz — APPROVED** [review]

**Review by alexander-schranz — CHANGES_REQUESTED** [review]
@Prokyonn UPGRADE.md missing

**Pull request merged**